tests/hooks/use-content-prediction.test.tsx

```
@@ -0,0 +1,45 @@
+
+    // Just verify the function exists
+    expect(typeof result.current.clearPrediction).toBe('function')
+
+    // Call it to ensure it doesn't throw
+    act(() => {
+      result.current.clearPrediction()
+    })
+
+    expect(result.current.prediction).toBeNull()
+  })
+}) 
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)